### ISSUE TITLE
Runtime exceptions on Flutter 1.22.0

### ISSUE THREAD

**Issue by kikuchy** [rrousselGit/flutter_portal#11]
`PortalEntry` causes some exceptions and it shows no overlaying widgets when using Flutter 1.22.0.

# Stacktraces

Trying `flutter_portal` with example code  https://github.com/rrousselGit/flutter_portal/blob/master/example/lib/date_picker.dart

```
════════ Exception caught by widgets library ═══════════════════════════════════════════════════════
'package:flutter/src/widgets/framework.dart': Failed assertion: line 6131 pos 12: 'slot == null': is not true.
The relevant error-causing widget was: 
  PortalEntry file:///Users/kikuchy/tmp/flutter_portal/example/lib/date_picker.dart:24:12
════════════════════════════════════════════════════════════════════════════════════════════════════
```

`slot` looks like `IndexedSlot` of `CompositedTransformTarget` at here https://github.com/rrousselGit/flutter_portal/blob/master/lib/src/portal.dart#L178

And using https://github.com/rrousselGit/flutter_portal/blob/master/example/lib/date_picker.dart , there are one more exception.

```
════════ Exception caught by widgets library ═══════════════════════════════════════════════════════
'package:flutter/src/widgets/framework.dart': Failed assertion: line 6131 pos 12: 'slot == null': is not true.
The relevant error-causing widget was: 
  LayoutBuilder file:///Users/kikuchy/tmp/flutter_portal/lib/src/portal.dart:183:18
════════════════════════════════════════════════════════════════════════════════════════════════════

════════ Exception caught by rendering library ═════════════════════════════════════════════════════
'package:flutter/src/widgets/layout_builder.dart': Failed assertion: line 161 pos 12: 'slot == null': is not true.
The relevant error-causing widget was: 
  LayoutBuilder file:///Users/kikuchy/tmp/flutter_portal/lib/src/portal.dart:183:18
════════════════════════════════════════════════════════════════════════════════════════════════════

════════ Exception caught by rendering library ═════════════════════════════════════════════════════
RenderBox was not laid out: _RenderLayoutBuilder#71d45 NEEDS-PAINT NEEDS-COMPOSITING-BITS-UPDATE
'package:flutter/src/rendering/box.dart':
Failed assertion: line 1785 pos 12: 'hasSize'
The relevant error-causing widget was: 
  Stack file:///Users/kikuchy/tmp/flutter_portal/lib/src/portal.dart:176:12
════════════════════════════════════════════════════════════════════════════════════════════════════
```

# Result of Flutter doctor

```
➜  example git:(master) ✗ flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 1.22.0, on Mac OS X 10.15.6 19G2021, locale ja-JP)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 12.0)
[✓] Android Studio (version 4.0)
[✓] IntelliJ IDEA Ultimate Edition (version 2020.2.2)
[✓] IntelliJ IDEA Community Edition (version 2019.2.3)
[!] VS Code (version 1.49.2)
    ✗ Flutter extension not installed; install from
      https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter
[✓] Connected device (1 available)

! Doctor found issues in 1 category.
```

**Comment by imtoori** [comment]
same issue here

**Comment by rrousselGit** [comment]
Fixed in 0.2.0
